src/component/SettingsForm/index.jsx

```diff
@@ -48,10 +50,29 @@ const SettingsForm = () => {
           </div>
         </div>
       ))}
-      <div className=" d-flex flex-row justify-content-center">
-        <Button variant="outline-dark" type="submit" className="w-25 mt-5">Save</Button>
+
+      <h5 className="mt-4 d-flex flex-row justify-content-center">Start Automaticaly Next period</h5>
+      <div className="mt-2 d-flex flex-row justify-content-center">
+        <BootstrapSwitchButton
+          onlabel="ON"
+          offlabel="OFF"
+          onstyle="outline-info"
+          offstyle="outline-dark"
+          width={100}
+          height={30}
+          checked={false}
```

Comment:
You should pass `shouldAutoStart` from App to here instead of hardcoding this to `false`. That way, if we change the default value in of shouldAutoStart to be `true`, then we wouldn't have to come and manually change it here too.

Reply:
Great idea, I want to ask in the ``settingsModal`` when I define the PropsType I write the ``shouldAutoStart``  ``isRequired`` but it appears to be the error that makes me give it a default value like
```
SettingsModal.defaultProps = {
  shouldAutoStart: false,
};
```
``TypeError: Cannot read property 'isRequired' of undefined``
I confused because in ``App.jsx`` I defined in ``useState`` the value of ``shouldAutoStart``.

Reply:
You fixed this, right? Because I'm not seeing that error. Let me know if you haven't fixed it yet and I can look into it with you.

Reply:
I do as you say in the first comment but as I say below your first comment the problem happen with me. The code work but still get this **Warning** 
 ``Warning: Failed prop type: SettingsModal: prop type `shouldAutoStart` is invalid; it must be a function, usually from the `prop-types` package, but received `undefined`.This often happens because of typos such as `PropTypes.function` instead of `PropTypes.func`.``